__tests__/app.test.js

```diff
@@ -76,4 +86,29 @@ it('returns a status code 404 when given an invalid article ID number', () => {
   })
 }
 )
-})
\ No newline at end of file
+})
+describe('GET /api/articles', () => {
+  it.only('returns the contents from the article table with all the correct properties with dates in descending order', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((response) => {
+    expect(Array.isArray(response.body.articles)).toBe(true);
+    response.body.articles.forEach(article => {
```

Comment:
Please remember to do a length check before a forEach. this will avoid false positives 👍